src/ui/components/Content.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { quizCreationStarted } from "../../state/actions/quizActions";
+import QuizesList from "./QuizesList";
+import { getQuizes } from "../../state/thunks/quizThunks";
+import { useEffect } from "react";
+import QuizForm from "./QuizForm";
+import Button from "../theme/Button";
+import Wrapper from "../theme/Wrapper";
+import Header from "../theme/Header";
+import Banner from "../theme/Banner";
+import TextBanner from "../theme/TextBanner";
```

Comment:
Cícero, não é 'obrigatório', mas tente organizar seus imports de uma forma que faça sentido: cada projeto é de uma forma, mas eu, particularmente, arrumaria assim:

```suggestion
import React from "react";
import { useSelector, useDispatch } from "react-redux";
import { quizCreationStarted } from "../../state/actions/quizActions";
import { getQuizes } from "../../state/thunks/quizThunks";
import { useEffect } from "react";
import QuizesList from "./QuizesList";
import QuizForm from "./QuizForm";
import Button from "../theme/Button";
import Wrapper from "../theme/Wrapper";
import Header from "../theme/Header";
import Banner from "../theme/Banner";
import TextBanner from "../theme/TextBanner";
```

Você pode organizar de ordem alfabética, enfim, de várias formas! O importante é manter um padrão em todos os arquivos!
Para arrumar em ordem alfabética, o prettier faz pra gente, se não souber como, me chame que eu te explico!

Para evitar muitos arquivos importados do mesmo lugar, você também pode usar o barrel file, mas não é obrigatório, só pra você saber que existe!

http://cangaceirojavascript.com.br/barrels-simplificando-importacoes-de-modulos/